content/en/altinity-kb-setup-and-maintenance/altinity-kb-zookeeper/zookeeper-cluster-migration-k8s-node-storage.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "ZooKeeper cluster migration when using K8s node local storage"
+linkTitle: "ZooKeeper cluster migration when using K8s node local storage"
+description: >
+  ZooKeeper cluster migration when using K8s node local storage
+---
+
+Describes how to migrate a ZooKeeper cluster when using K8s node-local storage such as static PV, `local-path`, `TopoLVM`.
+
+Requires HA setup (3+ pods).
+
+This solution is more risky than  due to reducing
```

Comment:
The phrase "than due to" was confusing. Are these alternatives?

Reply:
There was link that I somehow did not commit, thanks for catching